extensions/VisualEditor/ApiVisualEditor.php

```diff
@@ -47,6 +46,17 @@ public function getVersion() {
 		return $vers;
 	}
 
+	// Wikia change
+	/**
+	 * @protected
+	 * @description Simple helper to retrieve relevant host
+	 * @return String
+	 */
+	protected function getApiSource() {
+		$url = parse_url( wfExpandUrl( wfScript( 'api' ) ) );
```

Comment:
It seems like a unnecessary overhead to construct URL to api.php just so later you can extract from it the host part.